cmake/Modules/FindwxWidgetsTB.cmake

```diff
@@ -497,13 +497,13 @@ if(wxWidgets_FIND_STYLE STREQUAL "win32")
     if(MINGW)
       set(WX_LIB_DIR_PREFIX gcc)
     else()
-	  if(CMAKE_GENERATOR STREQUAL "Visual Studio 10 2010")
+	  if(CMAKE_VS_PLATFORM_TOOLSET STREQUAL "v100")

```

Comment:
The `WX_LIB_DIR_PREFIX` has a `c` inserted, so `vc100` for `CMAKE_VS_PLATFORM_TOOLSET` of `v100`.. I thought it'd be clearer to list all of the cases than do string manipulation